tools/src/main/scala/com/nvidia/spark/rapids/tool/qualification/QualificationArgs.scala

```diff
@@ -64,9 +64,10 @@ Usage: java -cp rapids-4-spark-tools_2.12-<version>.jar:$SPARK_HOME/jars/*
   val applicationName: ScallopOption[String] =
     opt[String](required = false,
       descr = "Filter event logs whose application name matches " +
-          "exactly or is a substring of input string. Regular expressions not supported." +
-          "For filtering based on complement of application name, use ~APPLICATION_NAME. i.e " +
-          "Select all event logs except the ones which have application name as the input string.")
+          "exactly or is a substring of input string. Application name as regular expression " +
```

Comment:
perhaps reword this to say something like:
Filter event logs by application name. The string specified can be a regular expression, substring, or exact match....

Reply:
Done